src/clj/web/lobby.clj

```diff
@@ -312,11 +330,14 @@
 
 (defmethod ws/-msg-handler :lobby/swap
   [{client-id :client-id
-    gameid :?data}]
+    {:keys [gameid side]} :?data}]
   (let [game (game-for-id gameid)
-        fplayer (first (:players game))]
-    (when (= (:ws-id fplayer) client-id)
-      (refresh-lobby-update-in gameid [:players] (partial mapv swap-side)))))
+        first-player (first (:players game))]
+    (when  (= (:ws-id first-player) client-id)
+      (if (> (count (:players game)) 1)
```

Comment:
```suggestion
      (if (< 1 (count (:players game)))
```
this is dumb but i prefer to keep all inequalities smaller on the left, helps keep things consistent (and i find it to be more natural).